LeapingGorilla.Testing/WhenTestingTheBehaviourOf.cs

```diff
@@ -175,8 +175,8 @@ private void PrepareMocksDependenciesAndItemUnderTest()
 				constructorArguments[index] = dep.Value;
 			}
 
-				// Create the item under test and load it into the test class
-			accessor[this, itemUnderTest.Name] = Activator.CreateInstance(itemUnderTest.Type, constructorArguments);

```

Comment:
I wasn't 💯 sure on this change but all the tests seem happy.  Seemed like a better approach then adding all the binding flags to the `Activator`